scRNA_qc/v1.1.0/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM rocker/tidyverse:4.3.2
+LABEL maintainer="Miguel Brown (brownm28@chop.edu)"
+LABEL description="Docker image to run scRNA QC. Install specific version of Seurat for downstream compatibility"
+
+ENV DEBIAN_FRONTEND=noninteractive
+ENV SCOOTER_COMMIT=cff289d
+ENV SEURAT_VERSION=4.3.0.1
+ENV SEURATOBJECT_VERSION=4.1.3
+ENV SEURAT_URL=https://cran.r-project.org/src/contrib/Archive/Seurat/Seurat_${SEURAT_VERSION}.tar.gz
+ENV SEURATOBJECT_URL=https://cran.r-project.org/src/contrib/Archive/SeuratObject/SeuratObject_${SEURATOBJECT_VERSION}.tar.gz
```

Comment:
I was thinking: rather than having these all at the top. I would make sense to declare them right before they are used. That way you can preserve layers when building and when one tool updates maybe you only need to rebuild the final layer rather than the whole thing.